### ISSUE TITLE
Feature: dependencies between npm packages

### ISSUE THREAD

**Issue by dherges** [ng-packagr/ng-packagr#1174]
## Type of Issue

```
[ ] Bug Report
[x] Feature Request
```

## Description

Feature: allow dependencies between packages (inter-package, different npm packages)

#### How To Reproduce

Library A:

```
- libs/
   |- a/package.json
   |- b/package.json
```

a depends on b, thus a would have b in its peerDependencies.

#### Expected Behaviour

Example could be (multiple "-p" parameters):

```bash
ng-packagr -p libs/a/package.json -p libs/b/package.json
```

#### Version Information

```
$ node_modules/.bin/ng-packagr --version
ng-packagr: 4.5.x
```

#### Related Issues

#379 
#911

**Comment by Itrulia** [comment]
Is there a way to atleast use the sourcefiles so we can run them in parallel?

I tried to change my tsconfig paths to

```
"@itrulia/error-reporting/*": [
    "libs/itrulia/error-reporting/*/src/public_api.ts"
],
```

from using the dist files but then i get the error

```
BUILD ERROR
Could not resolve entry
```

**Comment by alan-agius4** [comment]
Closing as out of scope
